apisix/plugins/routex.lua

```diff
@@ -0,0 +1,290 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core     = require("apisix.core")
+local upstream = require("apisix.upstream")
+local math     = math
+local ngx      = ngx
+
+
+local schema = {
```

Comment:
This cannot be read for a normal human.
You need to find a better way.

![image](https://user-images.githubusercontent.com/6814606/95599493-bc3c6400-0a83-11eb-94fb-e5f397b6f221.png)

Reply:
do you have any idea about how to fix this?

Reply:
We discussed, there is no better way.

Reply:
discussed? when? pls call me

Reply:
This `shcema` is created based on the data structure. To make it easier to understand, we need to modify the data structure.

Reply:
tip: variable reference

Reply:
ok, I know.